src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/library.cljs

```diff
@@ -0,0 +1,155 @@
+(ns org.broadinstitute.firecloud-ui.page.workspace.summary.library
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.input :as input]
+    [org.broadinstitute.firecloud-ui.common.modal :as modal]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.utils :as utils]))
+
+
+(defn- calculate-display-properties [library-schema]
+  (filter (comp not :hidden #(get-in library-schema [:properties %]))
+          (:propertyOrder library-schema)))
```

Comment:
Either way, it doesn't really matter.  In other places I'm assuming propertyOrder is a full list of all properties, which may or may not be a good assumption.